jest.config.js

```diff
@@ -1,26 +1,28 @@
 module.exports = {
-  testURL: 'http://localhost/',
-  moduleFileExtensions: ['js', 'jsx', 'ts', 'tsx', 'json', 'styl'],
-  setupFiles: ['<rootDir>/test/jestLib/setup.js'],
+  globals: {
+    __ALLOW_HTTP__: false,
+    __TARGET__: 'browser',
+    __SENTRY_TOKEN__: 'token',
+    cozy: {}
+  },
   moduleDirectories: ['src', 'node_modules'],
+  moduleFileExtensions: ['js', 'jsx', 'ts', 'tsx', 'json', 'styl'],
   moduleNameMapper: {
     '^lib/redux-cozy-client$': '<rootDir>/src/lib/redux-cozy-client',
     '\\.(png|gif|jpe?g|svg|css)$': '<rootDir>/test/__mocks__/fileMock.js',
-    // identity-obj-proxy module is installed by cozy-scripts
     '.styl$': 'identity-obj-proxy',
     '^cozy-client$': 'cozy-client/dist/index',
     '^test(.*)': '<rootDir>/test/$1'
   },
+  setupFilesAfterEnv: ['<rootDir>/test/jestLib/setup.js'],
+  testEnvironment: 'jsdom',
```

Comment:
nit: Do you know where we need DOM? Since jsdom has a setup cost for running the tests, if we know why we need it, maybe we can work on it to remove that dep?

Reply:
A test was failing without it, I'll see if I can make it work without jsdom

Reply:
let's create a specific issue about that point. It's a tricky point, that needs to be handled outside this PR.